framework/src/modules/chain/blocks/blocks.js

```diff
@@ -161,22 +140,19 @@ class Blocks extends EventEmitter {
 			1: block => this._receiveBlockFromNetworkV1(block),
 			2: block => this._receiveBlockFromNetworkV2(block),
 		};
-
-		this._processReceivedBlock = {
-			0: block => this._processReceivedBlockV1(block),
-			1: block => this._processReceivedBlockV1(block),
-			2: block => this._processReceivedBlockV2(block),
-		};
 	}
 
 	get lastBlock() {
-		return this._lastBlock;
+		// Remove receivedAt property..
+		const { receivedAt, ...block } = this._lastBlock;
+		return block;
 	}
 
 	get isActive() {
 		return this._isActive;
 	}
 
+	// TODO: Remove after fork
```

Comment:
what does remove after fork mean? Could you please elaborate the comment.

Reply:
When the new block version `2` is activated, the class property `_lastReceipt` will no longer be used or exposed, therefore we will be able to remove it.
Notice how `receiveBlockFromNetworkV2` now doesn't use this attribute whatsoever